scipy/interpolate/interpolate.py

```diff
@@ -343,13 +343,12 @@ class interp1d(_Interpolator1D):
         axis must be equal to the length of `x`.
     kind : str or int, optional
         Specifies the kind of interpolation as a string
-        ('linear', 'nearest', 'zero', 'slinear', 'quadratic', 'cubic',
-        'previous', 'next', where 'zero', 'slinear', 'quadratic' and 'cubic'
-        refer to a spline interpolation of zeroth, first, second or third
-        order; 'previous' and 'next' simply return the previous or next value
-        of the point) or as an integer specifying the order of the spline
-        interpolator to use.
-        Default is 'linear'.
+        ('linear', 'nearest', 'nearest-up', 'zero', 'slinear', 'quadratic',
+        'cubic', 'previous', 'next', where 'zero', 'slinear', 'quadratic'
+        and 'cubic' refer to a spline interpolation of zeroth, first,
+        second or third order; 'previous' and 'next' simply return the
+        previous or next value of the point) or as an integer specifying
+        the order of the spline interpolator to use. Default is 'linear'.
```

Comment:
This part needs a one-sentence explanation along the lines of ``"nearest-up" differs from "nearest" in that it ...'' or some such.

Reply:
Fixed.